build-system/tasks/serve.js

```diff
@@ -161,19 +204,12 @@ async function stopServer() {
 
 /**
  * Closes the existing server and restarts it
+ * @param {ServerOptionsDef=} serverOptions
  */
-async function restartServer() {
+async function restartServer(serverOptions = {}) {
```

Comment:
You're calling `restartServer()` on line 238 with `modeOptions`. Need to pass through the param here.

Reply:
added the param. thanks!